doc/languages-frameworks/index.xml

```diff
@@ -7,8 +7,9 @@
  </para>
  <xi:include href="agda.section.xml" />
  <xi:include href="android.section.xml" />
+ <xi:include href="bower.section.xml" />
  <xi:include href="beam.section.xml" />
- <xi:include href="bower.xml" />
+ <xi:include href="coq.xml" />
```

Comment:
Looks like you accidentally added old coq include back.

Reply:
And fixed this one. Thanks